samples/python/text2image/README.md

```diff
@@ -15,6 +15,10 @@ Users can change the sample code and play with the following generation paramete
 - (SD 1.x, 2.x only) Add negative prompt when guidance scale > 1
 - Apply multiple different LoRA adapters and mix them with different blending coefficients
 
+## Install OpenVINO™ GenAI and its dependencies
+
+The instruction is provided in the [`README`](../../../README.md).
+
```

Comment:
There was a feedback that instruction for openvino installation is missing in readme. 
So should we add this to all samples?

Reply:
CC @Wovchena 

Maybe we need to solve the issue in that PR https://github.com/openvinotoolkit/openvino.genai/pull/1048/files ? 
And show export-requirements.txt installation in model preparation section and deployment-requirements.txt in run section ?